pyblish_lite/window.py

```diff
@@ -627,6 +627,20 @@ def on_item_toggled(self, index, state=None):
 
         index.model().setData(index, state, model.IsChecked)
 
+        data = {"new_value": state, "old_value": not state}
+        if index.data(model.Type) == "instance":
+            instance = self.data["models"]["instances"].items[index.row()]
```

Comment:
Ohh, I should have related the commit I did for this to the issue #54. Duplicating the comment for better context: 

"I tried using ```index.data(model.Object)``` to fetch the original object, but this returns None. I'm resorting to ```self.data["models"]["instances"].items[index.row()]```"